website/client/src/components/groups/group.vue

```diff
@@ -58,10 +60,13 @@
             </div>
             <div
               v-if="!isParty"
-              class="col-4"
+              class="offset-1"
             >
               <div
                 class="item-with-icon"
+                tabindex="0"
+                role="button"
+                @keyup.enter="showMemberModal()"
```

Comment:
I think here it should call `showGroupGems `

Reply:
copy paste fail, thanks for catching!